README.md

```diff
@@ -1,27 +1,88 @@
 # Cybertown Revival
 
-Introduction about Cybertown Revival project and this repository
+This project is an attempt to resurrect and preserve Cybertown, a VRML based community from the 
+mid-90s/early-00s. This repository contains the entire codebase for the new platform, built by the community.
 
 
 ## How to Contribute
 
-tbc - forking, pull request, issues
+### As a Developer
 
+We always welcome others to help out with VRML and the Single Page Application (SPA) and API. Take a look 
+at our issue, fork this repository and start contributing. When you are ready, create a pull request, and we 
+will review your changes to be merged into the official master branch.
+
+### As a User
+
+Submitting bugs, feedback and commenting on issues is the best way for non-developers to help with the 
+project.
 
 ## Dev Stack
 
 * Node.js
-* MySQL
-* Nginx
 * Vue.js
 * Tailwind.css
+* MySQL
+* Nginx
 * socket.io
 * docker
+* VRML
 
 ## Development Environment Setup Instructions
 
-tbc - docker, npm, database population. don't forget installing the tools to install (ie npm, docker, node) and how to do it on Windows, Linux and Mac
+We utilise docker to manage the entire development environment and to make it easy to set up and run.
+
+### Requirements
+
+You will need to have the following already installed on your machine and a basic understanding in order to 
+run the development environment:
+
+* [node/npm][node]
+* [docker][docker-ce]
+
+You may also wish to install Docker for Desktop if you wish. For beginners, there are plenty of tutorials 
+and videos online on installation and the basics of node, npm and docker.
+
+### Initial Setup
+
+1. Clone this repository to your machine.
+2. Rename `spa/.env.example` to `spa/.env` and `api/.env.example` to `api.env`.

```

Comment:
I would probably split this into two separate steps for api and spa, and also include an `npm install` for each app for safe measure:
2. `cd spa/ && cp .env.example .env && npm install`
3. `cd ../api && cp .env.example .env && npm install`

Not sure if using raw shell commands as instructions is informative enough, I leave that up to you :)